nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -343,6 +343,15 @@ in
             (<literal>virtio</literal> or <literal>scsi</literal>).
           '';
         };
+
+      guestAgent =
```

Comment:
For consistency with other options I suggest calling this `guestAgent.enable` or `useGuestAgent`.
Also, spelling should be unified beween `guestAgent` for one option and `qemu-guest-agent` for the other.

Reply:
Ig uess I'll follow the pattern virtualboxGuest, hypvGuest with qemuGuest (https://nixos.org/nixos/options.html#guest)